### DISCUSSION TITLE
near-sdk-rs examples contribution

### DISCUSSION THREAD

**Discussion by On0n0k1** [near/near-sdk-rs#826] [Ideas]
Hello!

The examples are great! I like them a lot.

Just wanted to ask, if I made some loom videos explaining in detail each of the examples. Would you be interested in including them in the Readme?

Sincerely,

Lucas Lemos.

**Comment by willemneal** [comment]
Hey Lucas!

Currently there are no videos detailing the examples, but that is a great idea!  Another resource is https://near-sdk.io/ which goes over the same concepts as the examples.

**Comment by austinabell** [comment]
> Hello!
> 
> The examples are great! I like them a lot.
> 
> Just wanted to ask, if I made some loom videos explaining in detail each of the examples. Would you be interested in including them in the Readme?
> 
> Sincerely,
> 
> Lucas Lemos.

I think this is a great idea! Let us know if you need anything to complete this content :)

**Reply by On0n0k1** [reply]
Awesome! I think by the start of next month I can begin making the videos.
